introduction.md

```diff
@@ -4,3 +4,5 @@ INTRODUCTION
 * Code School Path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?